en/docs/learn/logging-in-to-salesforce-using-the-identity-server.md

```diff
@@ -42,26 +45,17 @@ Server credentials.
 			</Claim>
 			```
 
-			!!! note
-	
-				This file is checked only when WSO2 IS is starting for the first
-				time. Therefore, if you haven't configured this property at the time
-				of starting up the server for the first time, you will get errors at
-				the start up.
-	
-
-		3.  Open the `               <IS_HOME>/repository/conf/              `
-			`               identity/                               identity-mgt.properties               `
+		3.  Open the ` <IS_HOME>/repository/conf/identity/identity-mgt.properties `
 			file and set the following property to
-			`               true              ` .
+			`               true              `.
 
 			!!! info 
 				This step is required due to a known issue that prevents the
 				confirmation codes from being removed after they are used when email
-				usernames are enabled. This occurs because the '@' character (and
-				some special characters) are not allowed in the registry. To
+				usernames are enabled. This occurs because the '@' character and
+				some special characters are not allowed in the registry. To
 				overcome this issue, enable hashed usernames when saving the
-				confirmation codes by configuring the properties below.
+				confirmation codes by configuring the propert	ies below.
```

Comment:
please remove the extra spacing in "properties"

Reply:
corrected